dashboard/db/migrate/20160430111030_add_kind_to_survey_results.rb

```diff
@@ -0,0 +1,13 @@
+class AddKindToSurveyResults < ActiveRecord::Migration
+  def change
+    add_column :survey_results, :kind, :string, :after => :user_id
+    add_index :survey_results, :kind
+
+    reversible do |direction|
+      direction.up do
+        # On 2016-04-30, existing rows are the diversity survey from March 2016.
+        SurveyResult.update_all(kind: 'Diversity2016')
```

Comment:
My belief is that the add_column is (by default) reversed automatically be dropping the column. Assuming this belief is correct, this seems to be the desired behavior for downing the migration.